augur/augur_client.py

```diff
@@ -181,11 +181,9 @@ def filter_blocks(self, start_block, end_block, filter, handler,
     '''
     self._require_is_open()
     if (start_block < 0) or (end_block < 0) or (increment < 0):
-      raise IOError('Block numbers and increments should be greater than 0.')
+      raise IOError('Block numbers or increment cannot be less than 0.')
     if start_block > end_block:
       raise IOError('start_block must be less than the end_block.')
-    if start_block > self._ethereum_client.eth.blockNumber:
-      raise IOError('start_block has not been mined yet.')
     loop = asyncio.get_event_loop()
```

Comment:
Eliminate the use of the variable, a variable that is only ever used once is not worth using at all.